aws/resource_aws_iam_saml_provider_test.go

```diff
@@ -5,51 +5,42 @@ import (
 	"testing"
 
 	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/aws/awserr"
 	"github.com/aws/aws-sdk-go/service/iam"
+	"github.com/hashicorp/terraform/helper/acctest"
 	"github.com/hashicorp/terraform/helper/resource"
 	"github.com/hashicorp/terraform/terraform"
 )
 
-func TestAccAWSIAMSamlProvider_importBasic(t *testing.T) {
-	resourceName := "aws_iam_saml_provider.salesforce"
-
-	resource.Test(t, resource.TestCase{
-		PreCheck:     func() { testAccPreCheck(t) },
-		Providers:    testAccProviders,
-		CheckDestroy: testAccCheckIAMSamlProviderDestroy,
-		Steps: []resource.TestStep{
-			{
-				Config: testAccIAMSamlProviderConfig,
-			},
-
-			{
-				ResourceName:      resourceName,
-				ImportState:       true,
-				ImportStateVerify: true,
-			},
-		},
-	})
-}
-
 func TestAccAWSIAMSamlProvider_basic(t *testing.T) {
```

Comment:
super minor nit, but I think `SAML` should be capitalized like `AWS` and `IAM`

Reply:
Enabling `golint` would be an awesome goal (likely something for a 2.0+ cycle), but we'll need to look into ways we can enforce our own initialisms into it after fixing the current failures and ignoring purposeful things like the acceptance test function naming with underscores 😅 .  Without some form of enforcement, it could become a never ending project or very manual pain point during PR reviews. `SAML` unfortunately isn't one of the builtin initialisms and we'd also really want to add `AWS`, `EC2`, etc. 

I think we'll get there eventually, but we need to decide on a path forward that makes sense with the available tooling or work towards improving the tooling. 👍